src/sentry/models/rawevent.py

```diff
@@ -19,11 +21,12 @@ class RawEvent(Model):
     project = FlexibleForeignKey('sentry.Project')
     event_id = models.CharField(max_length=32, null=True)
     datetime = models.DateTimeField(default=timezone.now)
-    data = NodeField(
+    node_data = NodeField(
```

Comment:
to mirror the comment above: we might want to keep the field under the original name since it affects `bind_nodes`. Additionally if we do this change I believe we need to reflect this in a dummy db migration.

Reply:
Indeed, forgot to run migrations. I'll wait on a decision whether we should rename this or not. Regarding `bind_nodes`: If we do not rename this field, we'll have to come up with a new name for the accessor attribute and change every access to it. If we keep renaming it to `node_data` but don't want to change calls to bind_nodes, we'd have to look up the model's attribute name from the db_column (i.e. 'data' -> 'node_data'). There's probably a mapping somewhere in the meta model, but it sounds a bit strange to me tbh.